libraries/AP_LocationDB/tests/wscript

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env python
```

Comment:
Change the header just to this
```python3
#!/usr/bin/env python3
```

No need to set the encoding to utf-8 in python3.

Reply:
Done. Thanks.